example-configs/audit/rolebinding_viewers.yaml

```diff
@@ -8,6 +8,6 @@ roleRef:
   kind: ClusterRole
   apiGroup: rbac.authorization.k8s.io
 subjects:
-- name: 'system:serviceaccounts:audit'
-  kind: Group
-  apiGroup: rbac.authorization.k8s.io
+  - name: 'system:serviceaccounts:audit'
```

Comment:
how did we land on this style for formatting?

I asks because there may already be a norm in upstream k8s. e.g. See `Deployment` here:

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

If we want to be using the current norm (even if it suboptimal).

Reply:
This is due to the new yaml library upgrade.

Reply:
Not sure how that addresses my comment.

My question is about UX not library we choose to implement formatting. The style of formatting should not be changing based on some upstream library, it's an important part of the kpt UX.